tests/client.rs

```diff
@@ -18,7 +18,7 @@ fn create_client(config: &Config) -> Result<FileSystemClient> {
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
 async fn client_incrementally_fetches() -> Result<()> {
-    const RELEASE_COUNT: usize = 300;
+    const RELEASE_COUNT: usize = 10;
```

Comment:
Why is this release count reduced in this test?

Reply:
The test takes too long to run and times out with the changes. Also, kind of unrealistic test at `300` releases.